app/controllers/api/v1/events_controller.rb

```diff
@@ -16,6 +16,18 @@ def create
     end
   end
 
+  def update
+    @event = Event.find(params[:id])
+    if @event.update_attributes(event)
```

Comment:
you can use `update` instead of `update_attributes` as it's the [same thing](https://github.com/rails/rails/blob/5142d5411481c893f817c1431b0869be3745060f/activerecord/lib/active_record/persistence.rb#L254) but shorter.